src/Type/Php/BcMathStringOrNullReturnTypeExtension.php

```diff
@@ -78,11 +82,12 @@ public function getTypeFromFunctionCall(FunctionReflection $functionReflection,
 	 *
 	 * @param FuncCall $functionCall
 	 * @param Scope $scope
-	 * @return NullType|StringType|UnionType
+	 * @return NullType|Type|UnionType
```

Comment:
I guess we can just use `@return Type` here since both `NullType` and `UnionType` extend Type.
Or is there a way to say tt's a `String|AccessoryNumericStringType`, the intersect functions return a generic `Type` though

Reply:
Just Type should be here.